### PR TITLE
Change paratext casing

### PR DESCRIPTION
**Pull request by blakekrammes**

### CODE REVIEW

**Comment by CarsonF** [comment]
Is this field used in the UI?

**Comment by blakekrammes** [comment]
The plan was to have it in the v3 UI right?

**Comment by CarsonF** [comment]
Prob, but idk if it's there yet

**Comment by blakekrammes** [comment]
It's not there in v2, but this is how it's cased in our database and on paratext's website.

**Comment by blakekrammes** [comment]
Oh. You're wondering if this will break v3 frontend.

**Comment by CarsonF** [comment]
I don't disagree with the change, just want to make sure it's handling places since it's breaking.

**Comment by blakekrammes** [comment]
It's used in a couple components on the FE. I can change those.

**Comment by CarsonF** [comment]
Better yet. Add a deprecated field resolver so this isn't a breaking change

**Comment by CarsonF** [comment]
Example:
https://github.com/SeedCompany/cord-api-v3/blob/8a8d7cc5e1e87d5f934ee23ff85b9cae737a66ae/src/components/project/project.resolver.ts#L109-L115

**Comment by blakekrammes** [comment]
In addition to changing the FE or instead of?

**Comment by CarsonF** [comment]
Both. Doing it this way should be standard procedure going forward. This allows the API and consumers to be deployed independently without breaking anything.

**Comment by blakekrammes** [comment]
Do you then remove the deprecation code after?

**Comment by CarsonF** [comment]
Later yeah

**Comment by blakekrammes** [comment]
That look right?

**Review by CarsonF — DISMISSED** [review]
Yeah looks good. Technically we should support the old name on the input as well, but I'll let that slide as we aren't using it much yet.

**Comment by blakekrammes** [comment]
Where does that happen? In the dto?

**Comment by blakekrammes** [comment]
Like so?

**Comment by CarsonF** [comment]
Well yes in the DTO, but in the create/update mutations, not the output. The field you added to the DTO is the same as adding the field resolver, so it can be removed.

**Review by CarsonF — COMMENTED** [review]
Closer @blakekrammes. You accept the old form but then don't handle the different case in the service. They also should be nullable.

Again if we want to skip this on input I'm ok with it.

File: src/components/engagement/engagement.service.ts [hunk 1]
```diff
@@ -792,7 +792,7 @@ export class EngagementService {
           'communicationsCompleteDate',
           'startDateOverride',
           'endDateOverride',
-          'paraTextRegistryId',
+          'paratextRegistryId',
```
Comment (CarsonF) [inline src/components/engagement/engagement.service.ts:795]: Above where `changes` is defined:
```tsx
    const changes = pickBy({
      ...rest,
      paratextRegistryId: changes.paratextRegistryId || changes.paraTextRegistryId
      modifiedAt: DateTime.local(),
    }, val => val !== undefined);
```

**Pull request merged**